package.json

```diff
@@ -181,7 +182,7 @@
     "isomorphic-fetch": "^2.2.1",
     "jest": "^23.6.0",
     "jest-image-snapshot": "^2.7.0",
-    "jsdom": "^11.1.0",
+    "jsdom": "^16.2.0",
```

Comment:
this upgrade seems to have led to some sadness :( It was needed to make the RTL's `waitFor` helpers work.